Handle UFED other data of MessageExtraData

Currently in 4.3.0, unrecognized childs of `MessageExtraData` model are ignored. It is better to add this information in JSON preview of the related to the InstantMessage.

Example of the log handling the model:
```
2026-02-04 00:10:41     [ERROR] [datasource.ufed.UfedModelHandler]                      Unknown InstantMessageExtraData child 'MessageExtraData' => GenericModel (id=e1acfa17-f250-403b-8e20-0e9c6675fe8d). Ignoring...
2026-02-04 00:10:41     [ERROR] [datasource.ufed.UfedModelHandler]                      Unknown InstantMessageExtraData child 'MessageExtraData' => GenericModel (id=3ec8d849-3ba8-490c-ba7f-9b12b5c48020). Ignoring..
```

The xml chunk in report.xml.
```xml
<model type="ScrambledMessageData" id="e1acfa17-f250-403b-8e20-0e9c6675fe8d" deleted_state="Trash" decoding_confidence="High" isrelated="False" source_index="5219203" extractionId="0">
  <field name="Label" type="Label">
    <value type="Label"><![CDATA[Scrambled]]></value>
  </field>
</model>
```

--- a/iped-engine/src/main/java/iped/engine/datasource/ufed/UfedModelHandler.java
+++ b/iped-engine/src/main/java/iped/engine/datasource/ufed/UfedModelHandler.java
@@ -285,7 +285,7 @@ public class UfedModelHandler extends DefaultHandler {
                 } else if (child instanceof ReplyMessageData) {
                     extraData.setReplyMessage((ReplyMessageData) child);
                 } else {
-                    logger.error("Unknown InstantMessageExtraData child '{}' => {} (id={}). Ignoring...", fieldName, child.getClass().getSimpleName(), child.getId());
+                    extraData.getOtherData().add(child);
                 }
             } else if ("Position".equals(fieldName) && child instanceof Coordinate) {
                 message.setPosition((Coordinate) child);

--- a/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ufed/model/InstantMessageExtraData.java
+++ b/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ufed/model/InstantMessageExtraData.java
@@ -17,6 +17,7 @@ public class InstantMessageExtraData implements Serializable {
     private ForwardedMessageData forwardedMessage;
     private QuotedMessageData quotedMessage;
     private ReplyMessageData replyMessage;
+    private final List<BaseModel> otherData = new ArrayList<>();
 
     public List<MessageLabel> getMessageLabels() {
         return messageLabels;
@@ -44,5 +45,9 @@ public class InstantMessageExtraData implements Serializable {
 
     public void setReplyMessage(ReplyMessageData replyMessage) {
         this.replyMessage = replyMessage;
+    }
+
+    public List<BaseModel> getOtherData() {
+        return otherData;
     }
 }

